src/main/java/org/alfasoftware/soapstone/WebServiceInvoker.java

```diff
@@ -85,30 +86,47 @@ String invokeOperation(WebServiceClass<?> webServiceClass, String operationName,
 
     LOG.info("Invoking " + operationName);
     if (LOG.isDebugEnabled()) {
-      List<JsonNode> nodes = parameters.stream().map(WebParameter::getNode).collect(Collectors.toList());
-      LOG.debug("Parameters: " + nodes);
+      try {
+        List<JsonNode> nodes = parameters.stream().map(WebParameter::getNode).collect(Collectors.toList());
+        LOG.debug("Parameters: " + nodes);
+      } catch (Exception e) {
+        LOG.warn(String.format("Got a %s when try to trying the request", e.getClass().getSimpleName()));
+      }
     }
     if (LOG.isTraceEnabled()) {
-      List<String> prettyNodes = parameters.stream().map(parameter -> {
-        try {
-          return new ObjectMapper().writerWithDefaultPrettyPrinter().writeValueAsString(parameter.getNode());
-        } catch (JsonProcessingException e) {
-          throw new RuntimeException("JsonProcessingException while trying to pretty-print the request", e);
-        }
-      }).collect(Collectors.toList());
-      LOG.trace("Parameters: " + prettyNodes);
+      try {
+        List<String> prettyNodes = parameters.stream().map(parameter -> {
+          try {
+            return new ObjectMapper().registerModule(new JodaModule()).writerWithDefaultPrettyPrinter().writeValueAsString(parameter.getNode());
```

Comment:
Overlooked this last time, but I think this is the problem - we shouldn't be creating a new object mapper here - we should be using the one provided in the soapstone configuration.

Reply:
fixed in the last commit